service/service.go

```diff
@@ -24,21 +24,28 @@ import (
 	"go.opentelemetry.io/collector/config"
 	"go.opentelemetry.io/collector/consumer/consumererror"
 	"go.opentelemetry.io/collector/service/internal/builder"
+	"go.opentelemetry.io/collector/service/parserprovider"
 )
 
 // settings holds configuration for building a new service.
-type settings struct {
+type Settings struct {
 	// Factories component factories.
 	Factories component.Factories
 
-	// BuildInfo provides application start information.
-	BuildInfo component.BuildInfo
+	// ComponentSettings contains logger and build info configuration
+	ComponentSettings component.ComponentSettings
```

Comment:
Logger is constructed by the service, so probably better to keep only BuildInfo here.

Reply:
We can also do in a followup PR if we believe this is the way to go :)

Reply:
So, are you going to revert this change?

Reply:
Yes, I''l change that